examples/rust/custom_callback/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "custom_callback"
+version = "0.21.0-alpha.1+dev"
+edition = "2021"
+rust-version = "1.80"
+license = "MIT OR Apache-2.0"
+publish = false
+
+[[bin]]
+name = "custom_callback_viewer"
+path = "src/viewer.rs"
+
+[[bin]]
+name = "custom_callback_app"
+path = "src/app.rs"
+
+[features]
+default = []
+
+# Turn on if you want to share analytics with Rerun (e.g. callstacks on crashes).
+analytics = ["re_crash_handler/analytics", "re_viewer/analytics"]
+
+[dependencies]
+re_crash_handler = { path = "../../../crates/utils/re_crash_handler" }
+re_sdk_comms = { path = "../../../crates/store/re_sdk_comms", features = [
+  "server",
+] }
+re_viewer = { path = "../../../crates/viewer/re_viewer", default-features = false }
+re_log = { path = "../../../crates/utils/re_log" }
+re_error = { path = "../../../crates/utils/re_error" }
```

Comment:
I think we have re-exports for all of those in `rerun` - we should use those when possible instead of depending on more crates. That likely requires adding some features to `rerun`

Reply:
:point_up: I feel strongly about this. External users building Rerun extensions should never have to import anything but `rerun`.

Rerun is complicated enough without having to try and understand its internal architecture and dependencies :grimacing: